hazelcast-jet-core/src/main/resources/client-protocol-definition/Jet.yaml

```diff
@@ -21,9 +21,14 @@ methods:
           doc: ''
         - name: jobConfig
           type: Data
-          nullable: false
+          nullable: true
```

Comment:
This change means that old readers might have problems reading the data when the data is sent from the new Jet client. From the messages Ufuk wrote, I see that there is no compatibility guarantee between the Jet versions. Writing this to just double-check it.